tests/e2e/_test_enrichment_pipelines.py

```diff
@@ -0,0 +1,20 @@
+from sem_covid.entrypoints.ml_dags.enrich_dags import (enrich_eu_timeline_pipeline,
+                                                       enrich_ireland_timeline_pipeline, enrich_eu_cellar_pipeline)
+
+
+def test_eu_cellar_pipeline():
+    eu_cellar_pipeline = enrich_eu_cellar_pipeline
```

Comment:
this should be discovered and executed through DagBag (see  @fixture) and execute by `.run()`

`https://airflow.apache.org/docs/apache-airflow/1.10.12/_api/airflow/models/dag/index.html#airflow.models.dag.DAG.run`